content/go/CDN_services.go.md

```diff
@@ -0,0 +1,568 @@
+
+	// Create SoftLayer API session
+	sess := session.New(username, apikey)
+
+	service := services.GetNetworkCdnMarketplaceConfigurationMappingService(sess)
+	uniqueId := sl.Int(723231201263657)
+
+	receipt, err := service.VerifyDomainMapping(uniqueId)
+	if err != nil {
+		fmt.Printf("\n Unable to place order:\n - %s\n", err)
+		return
+	}
+
+	// Following helps to print the result in json format.
+	jsonFormat1, jsonErr1 := json.MarshalIndent(receipt, "", "     ")
+	if jsonErr1 != nil {
+		fmt.Println(jsonErr1)
+		return
+	}
+	fmt.Println(string(jsonFormat1))
+}
+```
+
+### StartDomainMapping
+
+```
+package main
+
+import (
+"encoding/json"
+"fmt"
+"github.com/softlayer/softlayer-go/services"
+"github.com/softlayer/softlayer-go/session"
+"github.com/softlayer/softlayer-go/sl"
+)
+
+func main() {
+	// SoftLayer API username and key
+	username := "set - me"
+	apikey := "set - me"
+
+	// Create SoftLayer API session
+	sess := session.New(username, apikey)
+
+	service := services.GetNetworkCdnMarketplaceConfigurationMappingService(sess)
+
+	uniqueId := sl.String("723231201263657")
+
+	receipt, err := service.StartDomainMapping(uniqueId)
+
+	if err != nil {
+		fmt.Printf("\n Unable to place order:\n - %s\n", err)
+		return
+	}
+
+	// Following helps to print the result in json format.
+	jsonFormat1, jsonErr1 := json.MarshalIndent(receipt, "", "     ")
+	if jsonErr1 != nil {
+		fmt.Println(jsonErr1)
+		return
+	}
+	fmt.Println(string(jsonFormat1))
+}
+```
+
+### StopDomainMapping
+
+```
+package main
+
+
+import (
+"encoding/json"
+"fmt"
+"github.com/softlayer/softlayer-go/services"
+"github.com/softlayer/softlayer-go/session"
+"github.com/softlayer/softlayer-go/sl"
+)
+
+func main() {
+	// SoftLayer API username and key
+	username := "set - me"
+	apikey := "set- me"
+
+	// Create SoftLayer API session
+	sess := session.New(username, apikey)
+
+	service := services.GetNetworkCdnMarketplaceConfigurationMappingService(sess)
+    // Input the uniqueId
+	uniqueId := sl.String("123456789")
+
+	receipt, err := service.StopDomainMapping(uniqueId)
+
+	if err != nil {
+		fmt.Printf("\n Unable to place order:\n - %s\n", err)
+		return
+	}
+
+	// Following helps to print the result in json format.
+	jsonFormat1, jsonErr1 := json.MarshalIndent(receipt, "", "     ")
+	if jsonErr1 != nil {
+		fmt.Println(jsonErr1)
+		return
+	}
+	fmt.Println(string(jsonFormat1))
+}
+```
+
+### ListDomainMappings
+
+```
+package main
+
+import (
+"encoding/json"
+"fmt"
+"github.com/softlayer/softlayer-go/services"
+"github.com/softlayer/softlayer-go/session"
+)
+
+func main() {
+	// SoftLayer API username and key
+	username := "set - me"
+	apikey := "set - me"
+
+	// Create SoftLayer API session
+	sess := session.New(username, apikey)
+
+	service := services.GetNetworkCdnMarketplaceConfigurationMappingService(sess)
+
+	receipt, err := service.ListDomainMappings()
+	if err != nil {
+		fmt.Printf("\n Unable to place order:\n - %s\n", err)
```

Comment:
Make this error more generic, since this isn't placeing an order.

fmt.Printf("\nAPI Error:\n - %s\n", err)